src/app/lib/provider.ts

```diff
@@ -1,6 +1,8 @@
-import { getDefaultProvider } from "ethers";
+import { getDefaultProvider, AlchemyProvider } from "ethers";
```

Comment:
not specific to this PR but man would this be awesome as a collection of viem public and wallet clients